lib/Minilla/Project.pm

```diff
@@ -455,6 +455,7 @@ sub cpan_meta {
         version        => $self->version,
         name           => $self->dist_name,
         prereqs        => $merged_prereqs,
+        optional_features => {},
```

Comment:
This makes META files always have optional_features field.
I don't think we need it if it is empty.

Reply:
I considered this.  It turns out that if it is empty, it doesn't get written into `META.json` (at least with the version of CPAN::Meta I tested with, 2.150010).  However, I can rewrite it to add this key only if non-empty, if you prefer?

Reply:
```
❯ perl -MCPAN::Meta\ 99999
CPAN::Meta version 99999 required--this is only version 2.150010.
BEGIN failed--compilation aborted.

❯ perl -Ilib script/minil dist --no-test

❯ tar xf Minilla-v3.1.0.tar.gz

❯ grep optional_features Minilla-v3.1.0/META.*
Minilla-v3.1.0/META.json:   "optional_features" : {},
Minilla-v3.1.0/META.yml:optional_features: {}
```

Reply:
Apologies.  I was looking only at the `META.json` created in the top-level directory, and had not realised that this file was generated differently for the tarball.